common/src/main/java/org/apache/celeborn/common/network/sasl/registration/RegistrationClientBootstrap.java

```diff
@@ -259,4 +267,34 @@ private static Map<PbAuthType, Set<String>> findSupportedSaslMechs(
     }
     return supportedMechs;
   }
+
+  @SuppressWarnings("NonInclusiveLanguage")
+  private static final Pattern MASTER_NOT_LEADER_EXCEPTION_PATTERN =
```

Comment:
Could the pattern be simplified?

Reply:
I think unfortunately given how the error message is, it has to be like this probably